docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerCloud.java

```diff
@@ -643,7 +643,7 @@ public boolean equals(Object o) {
 
     }
 
-    /* package */ boolean isSwarm() {
+    public boolean isSwarm() {
```

Comment:
no way, planned for removal.
swarm standalone should be replace with docker swarm mode as soon as we get some implementation working.

Reply:
Please don't !
Swarm mode is not designed for such use case, and some people (me included) may be happy with classic swarm, with no time/resources to switch over to swarm mode.

Reply:
I'm not saying we won't support swarm standalone anymore (it still make sense), just we will avoid specific code for this use case without some explicit added value.